application/frontend/web/index.php

```diff
@@ -1,16 +1,5 @@
 <?php
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'dev');
 
-require(__DIR__ . '/../../vendor/autoload.php');
-require(__DIR__ . '/../../vendor/yiisoft/yii2/Yii.php');
-require(__DIR__ . '/../../common/config/bootstrap.php');
-require(__DIR__ . '/../config/bootstrap.php');
+$application = require('load-app.php');
```

Comment:
ok...I was pulling out what was necessary for both the webapp and the tests.  This also allowed me to understand what exactly it was that the webapp needed that was different from the tests.  I was hoping to make them both the same.